lib/ansible/plugins/callback/__init__.py

```diff
@@ -263,6 +263,10 @@ def _clean_results(self, result, task_name):
                 # 'var' value as field, so eliminate others and what is left should be varname
                 for hidme in self._hide_in_debug:
                     result.pop(hidme, None)
+                # when using loop control, remove whatever is under loop_var
+                if 'ansible_loop_var' in result:
```

Comment:
We might want to do the same for  `ansible_index_var` variable too.

Reply:
good call, added that to the test playbook and found it show up. Pushing change now.

Reply:
actually, any var that is not the 'item' should be removed